src/types/index.ts

```diff
@@ -176,3 +176,12 @@ export interface Question {
   options: QuestionOption[];
 }
 
+/**
+ * Type describing a questionnaire in New Questionnaire Screen
+ */
+export interface Questionnaire {
```

Comment:
We should be able to get rid of the Questionnaire and Question type in this types file as we can use the types from the generated files. This also gives us better type safety and should make it easier to notice any mis-matches between the spec and the UI